calico_kubernetes/calico_kubernetes.py

```diff
@@ -37,7 +44,9 @@ def __init__(self):
         self.profile_name = None
         self.namespace = None
         self.docker_id = None
+
         self._datastore_client = DatastoreClient()
+        self._ipam_client = IPAMClient()
```

Comment:
IPAMClient inherits from DatastoreClient. So any calls you make through datastore_client can just be made through ipam_client. (right @caseydavenport ?)